js/main.js

```diff
@@ -20,4 +28,73 @@ $entryForm.addEventListener('submit', function (event) {
   data.entries.push(inputs);
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
+  addEntry(data.entries[data.entries.length - 1]);
+  $divHidden.setAttribute('class', 'hidden');
+  $divContainerLast.setAttribute('class', 'container');
+  $noEntries.setAttribute('class', 'hidden');
+});
+
+function addEntry(object) {
+  var newEntry = entryDOMTree(object);
+  $ul.prepend(newEntry);
+}
+
+function entryDOMTree(object) {
+  var liOut = document.createElement('li');
+
+  var divRow = document.createElement('div');
+  divRow.setAttribute('class', 'row');
+  liOut.appendChild(divRow);
+
+  var divColImg = document.createElement('div');
+  divColImg.setAttribute('class', 'column-half');
+  divRow.appendChild(divColImg);
+
+  var divColText = document.createElement('div');
+  divColText.setAttribute('class', 'column-half padding-right-0');
+  divRow.appendChild(divColText);
+
+  var img = document.createElement('img');
+  img.setAttribute('src', object.photoURL);
+  img.setAttribute('class', 'image margin-bot-25');
+  divColImg.appendChild(img);
+
+  var header = document.createElement('h1');
+  header.setAttribute('class', 'font-family margin-none');
+  divColText.appendChild(header);
+
+  var paragraph = document.createElement('p');
+  divColText.appendChild(paragraph);
+
+  header.textContent = object.title;
+  paragraph.textContent = object.notes;
+
+  return liOut;
+}
+
+window.addEventListener('DOMContentLoaded', function (event) {
+  if (data.entries.length === 0) {
+    $noEntries.setAttribute('class', 'row font-family justify-center');
+  }
+  for (var i = 0; i < data.entries.length; i++) {
+    var newEntry = entryDOMTree(data.entries[i]);
+    $ul.insertBefore(newEntry, $ul.childNodes[0]);
+  }
+  if (data.view === 'entry-form') {
+    $divHidden.setAttribute('class', 'container');
+    $divContainerLast.setAttribute('class', 'hidden');
+  }
+  if (data.view === 'entries') {
+    $divContainerLast.setAttribute('class', 'container');
+  }
```

Comment:
Instead of individual conditionals, why don't you loop over a list of all the view container elements, and check if the value of `data-view` on that element is equal to `data.view`, and if it is, show that view. Otherwise, hide that view.